### ISSUE TITLE
Elasticsearch pack doesn't work with https endpoints

### ISSUE THREAD

**Issue by pixelrebel** [StackStorm/st2contrib#575]
I have elastic search behind nginx proxy with SSL and http_auth.  The following command doesn't work with my `https` endpoint.

```
st2 run elasticsearch.indices.show host=elasticsearch.mydomain.net port=443 http_auth=stanley:210DAUhuasyH7ds&v use_ssl=true
```

It works over http on 9200, but not https on 443.  Looking at the traceback, it shows that it never tries https, so I'm assuming the use_ssl option is for shield?  

```
connection: getaddrinfo() argument 2 must be integer or string
WARNING:elasticsearch:GET http://elasticsearch.mydomain.net:443/ [status:N/A request:0.000s]
```

Any ideas?  Is this a issue that should be posted to the elasticsearch python API?

**Comment by pixelrebel** [comment]
There's [a simple fix](https://github.com/pixelrebel/st2contrib/blob/606c6d4c1fa38a79a6be9dbfd980fb8da66f25df/packs/elasticsearch/actions/lib/utils.py#L69) for this.  Changing from `host` to `hosts` parameter allows the st2 user to specify protocol, and it's backwards compatible with simple hostnames.  Now you can do this:
```
st2 run elasticsearch.indices.show host=https://elasticsearch.mydomain.net port=443 http_auth=stanley:210DAUhuasyH7ds&v use_ssl=true
```

Let me know if you want a PR, or I can hold off until you update for 2.4?

**Comment by lakshmi-kannan** [comment]
>or I can hold off until you update for 2.4?

If the fix also works for 2.4, please open a PR.

**Comment by LindsayHill** [comment]
Addressing this here: https://github.com/StackStorm-Exchange/stackstorm-elasticsearch/pull/6

Still need proper fix to move to 5.x.
